api/envoy/service/auth/v3/external_auth.proto

```diff
@@ -60,8 +60,20 @@ message DeniedHttpResponse {
   string body = 3;
 }
 
+// TODO: Should this be in the core API?
+message QueryParameterOption {
+  // The key of a the query parameter. Must be non-empty.
+  string key = 1 [(validate.rules).string = {min_len: 1}];
+
+  // The value of the query parameter. May be empty, e.g. for a query parameter such as `foo=`.
```

Comment:
Super nit. Prefer double-`.

```suggestion
  // The value of the query parameter. May be empty, e.g. for a query parameter such as ``foo=``.
```

Reply:
I ended up moving this to base.proto and removed the comment part that used `.